tfhe/docs/fundamentals/encrypt-data.md

```diff
@@ -13,12 +13,12 @@ use tfhe::{generate_keys, ConfigBuilder, FheUint8};
 fn main() {
     let config = ConfigBuilder::default().build();
 
-    let (client_key, server_key) = generate_keys(config);
+    let (client_key, _server_key) = generate_keys(config);
 
     let clear_a = 27u8;
     let clear_b = 128u8;
 
-    let a = FheUint8::encrypt(clear_a, &client_key);
-    let b = FheUint8::encrypt(clear_b, &client_key);
+    let _a = FheUint8::encrypt(clear_a, &client_key);
+    let _b = FheUint8::encrypt(clear_b, &client_key);
```

Comment:
let's keep the allow dead code in the tfhe/src/test_user_docs.rs otherwise the outside facing documentation will look very weird